sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -718,6 +718,7 @@
           <artifactId>maven-surefire-plugin</artifactId>
           <version>3.0.0-M3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-surefire-plugin;external_dependency} -->
           <configuration>
+            <runOrder>alphabetical</runOrder>
```

Comment:
This will be override by client setting. Storage has its own surefire setting.

Reply:
Maven by default merges configuration, since this isn't set in Storage it should use this value.